assets/app/scripts/controllers/deploymentConfig.js

```diff
@@ -53,6 +53,7 @@ angular.module('openshiftConsole')
       .get($routeParams.project)
       .then(_.spread(function(project, context) {
         $scope.project = project;
+        $scope.logContext = context;
```

Comment:
wanted to name it accordingly but we are already using this `logContext` for storing the context. eg:
https://github.com/openshift/origin/blob/master/assets/app/scripts/controllers/pod.js#L52
https://github.com/openshift/origin/blob/master/assets/app/scripts/controllers/deployment.js#L65

but will replace the name through whole console to make the name accurate